tests/unit_tests/dft/include/compute_tester.hpp

```diff
@@ -49,8 +49,8 @@ struct DFT_Test {
 
     enum class MemoryAccessModel { buffer, usm };
 
-    const std::int64_t size;
-    const std::int64_t conjugate_even_size;
+    const std::size_t size;
+    const std::size_t conjugate_even_size;
```

Comment:
I was wondering about this. We often use these sizes to initialize a vector or a buffer which expect a `size_t`. This would be an issue if we try to test negative sizes so using `size_t` looks safer to me.

Reply:
Sorry, I'm not sure I follow the logic there. How would we test negative sizes if we use an unsized type or is the point that you want to avoid that? (maybe just creating new tests/adding to descriptor tests would be easier)

Reply:
Sorry I meant that I think we should avoid testing negative sizes with this descriptor so I have a preference for using `size_t` here.

Reply:
Ok then, I can get with that. please resolve this comment.